python/src/tests/tasks/test_differential_expression.py

```diff
@@ -53,18 +53,10 @@ def test_throws_on_missing_parameters(self):
             DifferentialExpression()
 
     @responses.activate
-    def test_generates_correct_request_keys(self, mock_S3_get):
+    def test_throws_when_second_cellset_missing(self, mock_S3_get):
         MockS3Class.setResponse("one_set")
-        request = DifferentialExpression(self.get_request())._format_request()
-        assert isinstance(request, dict)
-
-        # all expected keys are in the request
-        expected_keys = [
-            "baseCells",
-            "backgroundCells",
-        ]
-
-        assert all(key in request for key in expected_keys)
+        with pytest.raises(Exception, match="fullfills the 2nd cell set"):
```

Comment:
If we are going to test that parameter existence is checked for, we shoudl also have a test that checks for the first parameter existence. 

Since that is not the objective of this ticket, could we add to [the other ticket we created](https://biomage.atlassian.net/browse/BIOMAGE-1694) for this a point on this?

Reply:
Sounds good, should I remove this then or just add a point to also add test for first parameter?